handlers/root.go

```diff
@@ -18,7 +18,14 @@ func Root(w http.ResponseWriter, r *http.Request) {
 }
 
 func getStatus(w http.ResponseWriter, r *http.Request) {
-	RespondWithJSONIndent(w, http.StatusOK, status.Get())
+
+	tokenContext := r.Context().Value("token")
+	if tokenContext != nil {
+		token := tokenContext.(string)
+		RespondWithJSONIndent(w, http.StatusOK, status.Get(token))
+	} else {
+		RespondWithJSONIndent(w, http.StatusInternalServerError, "Token missing in request context")
```

Comment:
Note that "Root" handlers is not authenticated:

```
		// swagger:route GET / Root
		// ---
		// Endpoint to get the status of Kiali
		//
		//     Produces:
		//     - application/json
		//
		//     Schemes: http, https
		// responses:
		//      500: internalError
		//      200: statusInfo
		{
			"Root",
			"GET",
			"/api",
			handlers.Root,
			false,
		},
```

So, that needs to be updated in the doc or let not authenticated scenarios.
Not sure if it's valid a token required in all cases.

Reply:
It has an empty string token in this case (which is an unauthenticated user). So the token value is set here (its set for all the endpoints that don't require authentication, see handlers.authentication HandleUnauthenticated)